src/sdk/facebook.js

```diff
@@ -56,6 +62,24 @@ const load = ({ appId, scope }) => new Promise((resolve) => {
   }
 })
 
+const handleInput = (input) => new Promise((resolve, reject) => {
+  let items = []
+  if (Array.isArray(input)) {
+    items = items.concat(input)
+  } else if (typeof input === 'string' && input) {
+    items = items.concat(input.split(','))
+  }
+
+  items = items.reduce((acc, item) => {
+    if (typeof item === 'string' && acc.indexOf(item) === -1) {
+      acc.push(item.trim())
+    }
+    return acc
+  }, []).join(',')
+
+  return resolve(items)
+})
+
```

Comment:
I would do it this way:

```js
const handleInput = (input, defaultInput) => {
  let result

  if (Array.isArray(input)) {
    result = defaultInput.concat(input)
  } else if (input && typeof input === 'string') {
    result = defaultInput.concat(input.split(','))
  }

  result = result.reduce((acc, item) => {
    if (typeof item === 'string' && acc.indexOf(item) === -1) {
      acc.add(item.trim())
    }

    return acc
  }, new Set())

  return [...result]
}
```

And call would be `facebookFields = handleInput(field, facebookFields)`